### PR TITLE
Inav: update page

### PR DESCRIPTION
**Pull request by Magrid0**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/inav.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Inav
+
+> An advanced log file viewer for the small-scale.
+> More information: <https://docs.lnav.org/en/latest/>.
+
+- To load and follow the system syslog file:
+
+`inav`
+
+- To load all of the files in /var/log:
```
Comment (kbdharun) [inline pages/linux/inav.md:10]:
```suggestion
- To load all of the files in `/var/log`:
```

File: pages/linux/inav.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Inav
+
+> An advanced log file viewer for the small-scale.
+> More information: <https://docs.lnav.org/en/latest/>.
+
+- To load and follow the system syslog file:
```
Comment (kbdharun) [inline pages/linux/inav.md:6]:
```suggestion
- Load and follow the system syslog file:
```

File: pages/linux/inav.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Inav
+
+> An advanced log file viewer for the small-scale.
```
Comment (kbdharun) [inline pages/linux/inav.md:3]:
```suggestion
> Advanced log file viewer for the console.
```

File: pages/linux/inav.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- To load and follow the system syslog file:
+
+`inav`
+
+- To load all of the files in /var/log:
+
+`inav /var/log`
+
+- To load all of the files in /var/log:
```
Comment (kbdharun) [inline pages/linux/inav.md:14]:
```suggestion
- Load all of the files in `/var/log`:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/inav.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Inav
+
+> An advanced log file viewer for the small-scale.
+> More information: <https://docs.lnav.org/en/latest/>.
+
+- To load and follow the system syslog file:
+
+`inav`
```
Comment (kbdharun) [inline pages/linux/inav.md:8]:
```suggestion
`lnav`
```

File: pages/linux/inav.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+- To load all of the files in /var/log:
+
+`inav /var/log`
```
Comment (kbdharun) [inline pages/linux/inav.md:12]:
```suggestion
`lnav /var/log`
```

File: pages/linux/inav.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- To load all of the files in /var/log:
+
+`make 2>&1 | inav -t`
```
Comment (kbdharun) [inline pages/linux/inav.md:16]:
```suggestion
`make 2>&1 | lnav -t`
```

File: pages/linux/inav.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Inav
```
Comment (kbdharun) [inline pages/linux/inav.md:1]:
```suggestion
# lnav
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
I just now noticed when trying to rename the file (after the `inav`, `lnav` confusion) that we already have it [here](https://github.com/tldr-pages/tldr/blob/main/pages/linux/lnav.md). Feel free to update, add contents the existing page.

**Comment by Magrid0** [comment]
> I just now noticed when trying to rename the file (after the `inav`, `lnav` confusion) that we already have it [here](https://github.com/tldr-pages/tldr/blob/main/pages/linux/lnav.md).

oh wait, it's an "L" not an "i", I hadn't noticed, my bad sorry for my mistakes.
I'll see if I find more examples.

**Review by blueskyson — APPROVED** [review]

File: pages/linux/inav.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+`lnav`
+
+- Load all of the files in `/var/log`:
+
+`lnav /var/log`
+
+- Load all of the files in `/var/log`:
+
+`make 2>&1 | lnav -t`
```
Comment (blueskyson) [inline pages/linux/inav.md:16]: Is there a Makefile to execute `make 2>&1`?

Comment (Magrid0) [reply pages/linux/inav.md]: I don't know, I pasted the example given in this [issue](https://github.com/tldr-pages/tldr/issues/9995)

File: pages/linux/lnav.md [hunk 10]
```diff
@@ -14,3 +14,7 @@
 - Validate the format of log files against the configuration and report any errors:
 
 `lnav -C {{path/to/log_directory}}`
+
+- Load all of the files in '/var/log':
```
Comment (kbdharun) [inline pages/linux/lnav.md:18]:
```suggestion
-  View logs of  the directory '/var/log':
```
I have a minor suggestion
> I don't think this example is necessary, but it is fine IG.

File: pages/linux/lnav.md [hunk 11]
```diff
@@ -14,3 +14,7 @@
 - Validate the format of log files against the configuration and report any errors:
 
 `lnav -C {{path/to/log_directory}}`
+
+- View logs of the directory '/var/log':
```
Comment (einverne) [inline pages/linux/lnav.md:18]: I think the first example includes the usage of this. There is no need to add this new example.

**Review by einverne — CHANGES_REQUESTED** [review]

File: pages/linux/lnav.md (hunk 11 above)
Comment (Magrid0) [reply pages/linux/lnav.md:18]: I'll remove it

Comment (einverne) [reply pages/linux/lnav.md:18]: Yes, I mean this example(line 18~20) is a specific usage of the example one. So we do not accept this  new update.

**Review by einverne — CHANGES_REQUESTED** [review]

**Pull request closed without merging**